cookiecutter.json

```diff
@@ -8,5 +8,6 @@
   "default_branch": "Name of the default branch, where the prod and staging ML resources are deployed from and the latest ML code is staged. Default: `main`",
   "release_branch": "Name of the release branch. The production jobs (model training, batch inference) defined in this stack pull ML code from this branch. Default: `release`",
   "read_user_group": "Display name of the user group to give read permissions to the ML jobs, integration test job runs, and machine learning resources created for this project. A group with this name must exist in both the staging and prod workspaces. Default: `users`",
+  "include_feature_store": false,
```

Comment:
It would be nice to have this parameter documented like the others - but requires some logic to replace the default value with the correct default & validate user input (since we'd need to make this a string parameter)

Reply:
I changed this to a choice variable in the meantime (as boolean support is not official yet.) . Is there no way to document these types of variable, hmm looking into it.

Reply:
Ah I see, yeah I guess no way to document choice variables, this is fine as is / clear enough I think - the alternative would have been to have a text variable where we document + validate that the user-supplied value is either "yes" or "no", which is arguably not much clearer